pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -338,8 +338,9 @@ func (s ShutdownStrategy) ShouldExecute(isOnExitPod bool) bool {
 	}
 }
 
+// +kubebuilder:validation:Type=array
```

Comment:
kubebuilder is too limited to correctly describe this type. We must choose a best option.